xcodeproj/internal/xcschemes/xcschemes_execution.bzl

```diff
@@ -317,27 +318,31 @@ def _write_schemes(
             _add_execution_action(
                 action,
                 action_name = action_name,
-                id = target.id,
+                id = target_id,
                 is_pre_action = FALSE_ARG,
                 scheme_name = scheme_name,
             )
 
     # buildifier: disable=uninitialized
     def _add_launch_target(launch_target, *, action_name, scheme_name):
         custom_scheme_args.add(launch_target.is_path)
+        target_id = None
```

Comment:
Nit: can you assign this in the `if`, to reduce a double assignment in the `else` path?